packages/components/index.js

```diff
@@ -12,8 +12,14 @@ module.exports = {
     return true;
   },
 
-  included: function (/* app */) {
+  included: function (app) {
     this._super.included.apply(this, arguments);
+    if (app.options && app.options['hds-code-block']) {
+      const options = app.options['hds-code-block'];
+      options.languages.forEach((language) => {
```

Comment:
[Example](https://github.com/froala/ember-froala-editor/blob/19fee2c9a2a0ec3a475a504773eb5ac88738fe6f/index.js#L123-L128) of how we could even validate that these languages exist